buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
```

Comment:
mavenLocal??

Reply:
We still use it to generate gradle dependency for the android project. Will be removed on the first release